packages/manager/src/features/CloudPulse/Widget/components/CloudPulseLineGraph.tsx

```diff
@@ -42,25 +32,9 @@ export const CloudPulseLineGraph = React.memo((props: CloudPulseLineGraph) => {
           <ErrorState errorText={error} />
         </Box>
       ) : (
-        <LineGraph
-          {...rest}
-          sxTableStyles={{
-            '& .MuiTable-root': {
-              border: 0,
-            },
-            backgroundColor: theme.bg.offWhite,
-            maxHeight: `calc(${theme.spacing(14)} + 3px)`,
-            minHeight: `calc(${theme.spacing(10)})`,
-            overflow: 'auto',
-            paddingLeft: theme.spacing(1),
-          }}
-          ariaLabel={ariaLabel}
-          data={data}
-          isLegendsFullSize={true}
-          legendRows={legendRows}
-        />
+        <AreaChart {...rest} />
```

Comment:
specific to this component, it seems it just another abstraction, do we need this now, what is benefits of this separate component